VAMobile/src/store/slices/appointmentsSlice.ts

```diff
@@ -142,6 +146,8 @@ export const initialAppointmentsState: AppointmentsState = {
     pastAllLastYear: {},
   },
   messagesLoading: false,
+  preloadComplete: false,
+  firstTimeLoading: true,
```

Comment:
Since we have `preloadComplete`, we can probably just use that. It serves the same purpose as `firstTimeLoading`